content/docs/04-clusters/06-cluster-management/02.5-certificate-management.md

```diff
@@ -0,0 +1,421 @@
+
+This reference page focuses on how to renew the PKI certificates through Palette. You have two options for how you can renew the cluster PKI certificates:
+
+  <br />
+
+  * Automatic Certificate Renewal
+
+
+  * Manual Certificate Renewal
+
+
+
+<InfoBox>
+
+Certificates created by kubeadm expire after 365 days. The Root Certificate Authority (CA) is valid for 3652 days or ten years.
+
+</InfoBox>
+
+
+You can learn more about each option in the following sections. 
+
+
+
+# Automatic Certificate Renewal
+
+When you upgrade the control plane on a cluster, the PKI certificates are automatically updated. Upgrading a cluster's Kubernetes version, whether a minor patch or a major release, results in renewed PKI certificates. This is the method that requires the least user actions when it comes to renewing PKI certificates. We recommend regularly updating your clusters to stay up-to-date with security fixes and best practices. By keeping your host cluster up-to-date, you prevent the scenario of PKI certificates from reaching their one-year expiration date.
+
+<br />
+
+<InfoBox>
+
+You can upgrade the Kubernetes version of a cluster by updating the Kubernetes layer of the [cluster profile](/cluster-profiles) and applying the cluster profile update to the cluster.
+
+</InfoBox>
+
+
+# Manual Certificate Renewal
+
+You can renew the cluster PKI certificates on-demand using the Palette API or the Palette User Interface (UI). When you manually trigger a certificate renewal action, the action will result in cluster nodes becoming repaved. Palette will scale up the cluster control plane nodes count and deploy new nodes with new PKI certificates auto-generated by kubeadm. Once the new control plane nodes are active and available, Palette will remove the previous control plane nodes. The worker nodes renew once the control plane nodes are updated and available.
+
+<br />
+
+<WarningBox>
+
+A manual renewal of cluster PKI certificates will result in all cluster nodes becoming repaved.
+
+</WarningBox>
+
+
+Review the following sections to learn more about manually renewing the cluster PKI certificates. Choose the preferred update method, API, or through the Palette UI.
+
+
+
+<br />
+
+<Tabs>
+
+<Tabs.TabPane tab="UI" key="ui">
+
+
+Use the following steps to update the cluster PKI certificates through the Palette UI.
+
+
+## Prerequisites
+
+- A host cluster deployed.
+
+
+- Access to the host cluster.
+
+## Renew Cluster Certificate
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+
+2. Navigate to the left **Main Menu** and select **Clusters**.
+
+
+3. Select the host cluster you want to renew the PKI certificates.
```

Comment:
```suggestion
3. Select the host cluster to renew its PKI certificates.
```

Reply:
If you want to keep "you want to", it's more correct to say the following, but then it sounds kind of formal:

"Select the host cluster for which you want to renew the PKI certificates."